packages/table/style.ts

```diff
@@ -40,3 +40,9 @@ export const hideScrollbarCss = css`
     display: none;
   }
 `;
+
+export const rowHoverCss = css`
+  background-color: ${greyLightLighten5};
+  mix-blend-mode: multiply;
+  will-change: left;
```

Comment:
Adding `will-change` is to fix a rendering bug in Chrome and Safari. The value has to be something that does not create stacking context, otherwise it will not work in Safari.

If you're curious for a demo of the bug this fixes, we can do a screenshare